config/locales/en.yml

```diff
@@ -478,12 +478,20 @@ en:
       label: Update
       field_help_texts:
         email: Please enter a valid email address if you would like to change your current one
+        contact_number: Please enter a valid UK phone number
         password: At least 8 characters. Leave blank if you don't want to change your password.
         password_confirmation: Leave blank if you don't want to change your password
         postcode: Please enter a valid UK postcode if would like to update your current one
         current_password: We need your current password to confirm your changes
+      contact_number:
+        label: Contact number (optional)
+      current_password: Contact details (required to update settings)
+      opt_in_for_research:
+        title: I accept to be contacted for research purposes
+      password:
+        title: Change my password(Optional)
```

Comment:
This would be a touch easier to read if the order of the keys was the same in both the Welsh and English files.